src/solver/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ add_subdirectory(simulation)
 add_subdirectory(ts-generator)
 add_subdirectory(utils)
 add_subdirectory(variable)
+add_subdirectory(optim)

```

Comment:
the idea was to avoid interaction between the old and the new codes, so we created this new namespace. yes, I think "optimisation" will disappear at some point

Reply:
Changing **solver/optim**  into **solver/modeler** would be quick and make things more clear.

Reply:
I agree

Reply:
Done